crawler/cmd/crawl.go

```diff
@@ -25,7 +26,19 @@ var crawlCmd = &cobra.Command{
 			if err != nil {
 				log.Fatal(err)
 			}
-			publishers = append(publishers, readWhitelist...)
+
+		Publisher:
+			for _, publisher := range readWhitelist {
+				for _, org := range publisher.Organizations {
+					if orgs[org] {
```

Comment:
I think we should do the same with `repo` key, wdyt @bfabio ?

Reply:
On principle I agree, but I tried to keep it simple and straightforward for our main issue: we may have duplicates between onboarding and the publishers added in the repo because the onboarding is user driven and doesn't check if the organization is already present in the YAML files (and that's ok and it's supposed to work).

That can happen anytime, but it's just for whole organizations, and if if does happen we give precedence to the publisher that just onboarded and eventually remove it from the YAML files in the repo. AFAIK there's no way for someone to onboard a specific repo.

The other side of things, and that's where you can also add single repos to the YAML files, I think is reasonably safe because it's A) manually reviewed and B) less frequent.

Checking for duplicate repos would be more expensive as we'd need to check both repos and organizations containing the URL. Also, I'm not sure there are no corner cases where https://host/foo/bar` is not in `https://host/foo`.

Reply:
@bfabio it makes sense 👍